data/mods/Xedra_Evolved/mapgen/overmap_special.json

```diff
@@ -283,5 +283,25 @@
     "occurrences": [ 5, 100 ],
     "flags": [ "UNIQUE", "WILDERNESS" ],
     "spawns": { "group": "GROUP_GENIUS_HOMULLUS", "population": [ 10, 30 ], "radius": [ 1, 3 ] }
+  },
+  {
+    "type": "overmap_special",
+    "id": "Vampire Hunting Lodge",
```

Comment:
I'm not sure where this entry got migrated from, but a big audit PR like this that's focused on mapgen weights should probably not also be doing this kind of cleanup in other files. It's just too much to track and check for errors.

Reply:
It's not migrating from anywhere, the vampire lodge maps currently spawn as part of the regular hunting lodge but as far as I can tell whoever did it thought they replace the vanilla maps bc currently you can get the vanilla basement with the vampire ground floor etc so I just made it it's own special so the whole thing spawns together. This is kind of weight related but I can pull it out to it's own PR if you'd still prefer that. (It won't need id migration or anything bc the old ids are still in use by vanilla)

Reply:
Yep, I thought I'd fixed that but apparently never did.  Thanks for catching this!